sql/telemetry/origin_content_blocking/view.sql

```diff
@@ -0,0 +1,28 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod.telemetry.origin_content_blocking` AS
+SELECT
+  submission_date,
+  batch_id,
+  origin,
+  aggregate
+FROM
+  (
+    SELECT
+      *,
+      ROW_NUMBER() OVER (
+        PARTITION BY
+          submission_date,
+          batch_id,
+          origin
+        ORDER BY
+          TIMESTAMP DESC
+      ) AS insert_rank
```

Comment:
how many rows does this dataset have? this method is known to have performance issues at high volume. it sounds like it's not much, but i want to clarify.

Reply:
At the moment, it's 3m rows, or about 20mb scanned for this view.